ccswbs/doorknob
# update home story

# Summary of changes
Update the story block on the home page with a new quote and image.

## Frontend
- updated home-story.yml
- update homeStory.js (change class on blockquote from display-4 to display-5)
- added image asha-edwin.png

[x] My changes are accessible (at minimum WCAG 2.0 Level AA)
[x] My changes are responsive and appear as expected on mobile and desktop views.

## Backend
N/A

# Test Plan

1. Go to preview link
2. Scroll to bottom of page
3. See new changes
